pkg/reconciler/parallel/parallel.go

```diff
@@ -172,33 +174,29 @@ func (r *Reconciler) reconcileChannel(ctx context.Context, channelResourceInterf
 	channelable, ok := c.(*duckv1.Channelable)
 	if !ok {
 		logger.Errorw("Failed to convert to Channelable Object", zap.Any("channel", channelObjRef), zap.Error(err))
-		return nil, fmt.Errorf("Failed to convert to Channelable Object: %+v", c)
+		return nil, fmt.Errorf("failed to convert to Channelable Object: %+v", c)
 	}
 	return channelable, nil
 }
 
 func (r *Reconciler) reconcileBranch(ctx context.Context, branchNumber int, p *v1.Parallel) (*messagingv1.Subscription, *messagingv1.Subscription, error) {
 	filterExpected := resources.NewFilterSubscription(branchNumber, p)
-	filterSubName := resources.ParallelFilterSubscriptionName(p.Name, branchNumber)
-
-	filterSub, err := r.reconcileSubscription(ctx, branchNumber, filterExpected, filterSubName, p.Namespace)
+	filterSub, err := r.reconcileSubscription(ctx, branchNumber, filterExpected)
 	if err != nil {
 		return nil, nil, err
 	}
 
 	expected := resources.NewSubscription(branchNumber, p)
-	subName := resources.ParallelSubscriptionName(p.Name, branchNumber)
```

Comment:
`subName` generates the same name that was used at `expected`. No need for that second call.